### PR TITLE
[4.0] Moving System & API Tests to main repo

### PR DESCRIPTION
**Pull request by Hackwar**
This moves the tests from joomla/test-api and joomla/test-system to the main repo. I also cleaned up some stuff.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -34,7 +34,6 @@ steps:
       - composer install --no-progress --no-suggest
       # needed for system testing
       - composer update joomla/test-system --no-progress --no-suggest
```
Comment (SniperSister) [inline .drone.yml]: looks obsolete, just like the other two update calls

Comment (Hackwar) [reply .drone.yml]: yes, will remove them shortly. Just have to find out why the tests fail right now...

**Review by SniperSister — APPROVED** [review]
Looks good to me, thank you :)

**Comment by SniperSister** [comment]
@Hackwar can you resolve the merge conflict in the drone yaml?

**Comment by Hackwar** [comment]
Fixed it. Can we merge that then now? ;-)

**Comment by SniperSister** [comment]
Merged! Thanks!

**Comment by Hackwar** [comment]
Thank you!

**Pull request merged**